asciidoc/getting_started/next/init.adoc

```diff
@@ -0,0 +1,65 @@
+== @ensono-stacks/next:init
```

Comment:
Could make the child pages collapsible. I've done this for cypress and playwright just to make it like for like with the current implementation.